tla-bmcmt/src/main/scala/at/forsyte/apalache/tla/tracee/DrivenTransitionConstructor.scala

```diff
@@ -0,0 +1,46 @@
+package at.forsyte.apalache.tla.tracee
+
+import at.forsyte.apalache.tla.lir.{NameEx, TlaEx}
+import at.forsyte.apalache.tla.types.tla._
+import at.forsyte.apalache.tla.lir.TypedPredefs.TypeTagAsTlaType1
+import at.forsyte.apalache.tla.lir.transformations.TransformationTracker
+import at.forsyte.apalache.tla.lir.transformations.standard.ReplaceFixed
+
+/**
+ * @author
+ *   Jure Kukovec
+ */
+class DrivenTransitionConstructor(
+    tracker: TransformationTracker,
+    exs: Map[String, TlaEx]) {
```

Comment:
Could you add a doccomment on this parameter?

Reply:
Sure